src/hourly-weather.ts

```diff
@@ -321,6 +326,30 @@ export class HourlyWeatherCard extends LitElement {
     return temperatures;
   }
 
+  private getWind(forecast: ForecastSegment[], numSegments: number, offset: number, speedUnit: string): SegmentWind[] {
+    const wind: SegmentWind[] = [];
+    for (let i = offset; i < numSegments + offset; i++) {
+      const fs = forecast[i];
+      let speed = '-';
+      let dir = '';
+      if (fs.wind_speed > 0) {
+        speed = `${Math.round(fs.wind_speed)} ${speedUnit}`.trim();
+        dir = this.formatWindDir(fs.wind_bearing);
+      }
+      wind.push({
+        hour: this.formatHour(new Date(fs.datetime), this.hass.locale),
+        windSpeed: speed,
+        windDirection: dir
+      })
+    }
+    return wind;
+  }
+
+  private formatWindDir(degrees: number): string {
+    const directions = ['N','NNE','NE','ENE','E','ESE','SE','SSE','S','SSW','SW','WSW','W','WNW','NW','NNW','N'];
```

Comment:
Can you make these localizable? Other languages may use different cardinal direction names.

Reply:
Is there a clean way to make an array of strings localizable, or am I best to add a separate language string for each direction and compose those into an array to use them?

Reply:
Probably best to make a separate object of loc strings in the locale jsons (e.g. direction.n) and then compose into an array at runtime -- ideally memoized into a member variable so you only have to compose it once as long as language doesn't change. I do that with the condition strings too I think, if you want to follow that pattern.

Reply:
I've used the condition strings as a guide, but deviated a little so that the member variable is the array (so there doesn't need to be an intermediate variable / another step to get to the array).  Does that look reasonable to you?